src/renderer/shared/core/types/stake.ts

```diff
@@ -19,3 +20,13 @@ export const enum RewardsDestination {
   RESTAKE = 'restake',
   TRANSFERABLE = 'transferable',
 }
+
+export type NominatorInfo<T = Account> = {
```

Comment:
Let's move it to `pages/Staking/Overview/common/types.ts` because it has value only for staking
`shared/core/types` contains base domain types, `NominatorInfo` is a composite type that might change in future if we rewrite Staking dashboard 😌